src/physics/utilities/equation_solver.cpp

```diff
@@ -38,6 +38,169 @@ EquationSolver::EquationSolver(MPI_Comm comm, const LinearSolverParameters& lin_
   }
 }
 
+namespace detail {
```

Comment:
I think this is fine for now, but it might be worthwhile to pull in a real JSON parser sooner rather than later. I can imagine needing one for other applications. I think there is already one in either Conduit or Axom. Thoughts @white238 ?

Reply:
One is actually already included in mfem: https://github.com/mfem/mfem/blob/master/fem/picojson.h

Reply:
I'm not sure if picojson is flexible enough for us, it seems mostly focused on parsing/accessing and not the ability to add/manipulate objects.

Conduit has a JSON library that we may not want to use right now, but Axom (I think) will have a different one soon for Inlet.